lib/res/formatters/rspec.rb

```diff
@@ -65,11 +65,11 @@ def example_pending(pending_example_notification)
         set_status(result, test.metadata[:location], status)
       end
 
-        def add_result(test)
-       return { 
-          "type": "Rspec::Test",
-          "name": test[:description],
-          "urn": test[:location],
+      def add_result(test)
+        {
+          'type' => 'Rspec::Test',
```

Comment:
If you modify your pull request as suggested, which will make it work with Ruby 2.1, I will merge it. Thanks.